internal/langserver/handlers/hooks_module.go

```diff
@@ -119,3 +120,22 @@ func updateDiagnostics(ctx context.Context, notifier *diagnostics.Notifier) stat
 		}
 	}
 }
+
+func refreshCodeLens(ctx context.Context, clientRequester session.ClientCaller) state.ModuleChangeHook {
+	return func(oldMod, newMod *state.Module) {
+		oldOrigins, oldTargets := 0, 0
+		if oldMod != nil {
+			oldOrigins = len(oldMod.RefOrigins)
+			oldTargets = len(oldMod.RefTargets)
+		}
+		newOrigins, newTargets := 0, 0
+		if newMod != nil {
+			oldOrigins = len(newMod.RefOrigins)
```

Comment:
```suggestion
			newOrigins = len(newMod.RefOrigins)
```
This might be a copy paste mistake?

Reply:
Ooops - yes! 🤦🏻‍♂️  Thanks for catching it!